astrodb_utils/sources.py

```diff
@@ -127,6 +129,23 @@ def find_source_in_db(
     if len(db_name_matches) == 1:
         db_names = db_name_matches["source"].tolist()
         logger.debug(f"One match found for {source}: {db_names[0]}")
+
+        # check coords of the match make sense
+        coord_check = (
+            ra
+            and dec
+            and np.isclose(ra, db_name_matches[ra_col_name][0])
```

Comment:
Do we want to allow for users to supply a tolerance level for matching? `np.isclose` has two threshold settings we could consider. It's possible the defaults are fine but they might be too specific for most use-cases.

Reply:
A possible approach is to calculate what is 1 arcsec in degrees and provide that as the absolute tolerance (atol) in np.isclose (by default this is 1e-08 which is very small)

Reply:
great catch!  I switched to using `astopy.units.isclose()` and used the search_radius as an absolute tolerance.  (Also turned search_radius into a quantity.)